### ISSUE TITLE
Loss of focus is observed while navigating from ".NET Framework/ .NET Core " drop-down to "Create Workbook" button by tabbing in "Create a New workbook" window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#326]
### Pre-Requisite:
Follow the instructions for installing the app from 
https://docs.microsoft.com/en-us/xamarin/tools/workbooks/install?tabs=windows 

### Steps to Reproduce

1. Launch Xamarin Workbooks application.
2. Select the working platform to "Console".
3. Navigate from ".NET Framework/ .NET Core" drop-down to "Create Workbook" button using Tab.

### Expected Behavior
Should be able to set focus on "Create Workbook" button with 1 Tab press from " .NET Framework/ .NET Core" drop-down.
### Actual Behavior
Loss of focus is observed while navigating from ".NET Framework/ .NET Core " drop-down to "Create Workbook" button.
### Note: (Inspect Observation)
Forward Navigation: Inspect focus remains 2 times on ".NET Framework/.NET Core" drop-down button
Backward Navigation: Inspect focus remains 2 times on "Create Workbook" button.
### User Impact
People with visual impairments can become disoriented when keyboard navigation takes focus somewhere unexpected or when they cannot easily find the content surrounding an interactive element.
### Environment
Windows : Version 1803 (OS Build : 17134.48)
Xamarin Version : 1.4(master/b3f92f9)
### Screenshots:
![issue 2](https://user-images.githubusercontent.com/22956130/40480531-7329ae0e-5f6c-11e8-9425-93bed20aa55a.png)

Inspect Screenshots:
![screenshot 54](https://user-images.githubusercontent.com/22956130/40480720-fc71c9a8-5f6c-11e8-8385-55011b0ee224.png)

![screenshot 57](https://user-images.githubusercontent.com/22956130/40480728-019868ce-5f6d-11e8-94a9-e633c584343b.png)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS2.4.3

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
